### PR TITLE
Creative project

### PR DESCRIPTION
**Pull request by franklynr92**
Added links for creative project new file location

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 Oftentimes I read these pages only to get the minimum amount of information for a required assignment, and then I moved on. I've collected them here for further study.
 
+#### Activity C - CSS Part 1 & 2; Web Design Principles Part 1 & 2;
+[Exercise Part 1](https://github.com/JaredEzz/software-engineering-resources/blob/master/Activity_C_Folder/%F0%9F%92%BB%20Activity%20C1_%20Learning%20CSS%20Part%201.pdf)
```
Comment (JaredEzz) [inline README.md]: you need two spaces after each line to make it a new line on .md

File: README.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+[Exercise Part 2](https://github.com/JaredEzz/software-engineering-resources/blob/master/Activity_C_Folder/%F0%9F%92%BB%20Activity%20C2_%20Learning%20CSS%20Part%202.pdf)
```
Comment (JaredEzz) [inline README.md]:
```suggestion
[Exercise Part 2](https://github.com/JaredEzz/software-engineering-resources/blob/master/Activity_C_Folder/%F0%9F%92%BB%20Activity%20C2_%20Learning%20CSS%20Part%202.pdf)  
```

Comment (JaredEzz) [reply README.md]: something like this

**Review by JaredEzz — COMMENTED** [review]
looks good, just a formatting fix

**Comment by franklynr92** [comment]
Added spacings to make lines on md file

**Pull request merged**